packages/runtime/src/dataViews/DataViewExpander.ts

```diff
@@ -1641,9 +1598,8 @@ export class DataViewExpander {
             attributeMap: attributesByType,
             items: expandedAttributes,
             nameMap: stringByType,
-            changes: changes,
-            changeCount: changes.length,
-            templateId: relationship.template.id
+            templateId: relationship.template.id,
+            auditLog: relationship.auditLog
```

Comment:
I fear the auditLog might need to be expanded also? E.g. there are addresses within which could be expanded

Reply:
IDK - don't know how exactly the DVO is used. ping @jkoenig134

Reply:
yeah, would be great if we expand addresses in the audit log.

Reply:
which parts of the standard peer address expansion do you need? A full expansion would cause a loop peerAddress -> relationship -> peerAddress

Reply:
as below, can be postponed. We don't need it in the App.